shared/src/business/utilities/DateHandler.test.ts

```diff
@@ -740,4 +743,59 @@ describe('DateHandler', () => {
       expect(result).toBe(true);
     });
   });
+
+  describe('ISO date range tests', () => {
+    const dcz = { zone: USTC_TZ };
+
+    //should return date range if given partial iso start date
+    it('should return date range if given partial iso start date', () => {
+      const start = '2002-11-01';
+      const expectedStart = DateTime.fromISO(start, dcz).toUTC().toISO();
+      const expectedEnd = DateTime.fromISO(start, dcz)
+        .endOf('day')
+        .toUTC()
+        .toISO();
+      const range = normalizeIsoDateRange(start, undefined);
+      expect(range.start).toBe(expectedStart);
+      expect(range.end).toBe(expectedEnd);
+    });
+
+    //should return date range if given two partial iso dates
+    it('should return date range if given two partial iso dates', () => {
+      const start = '2002-11-01';
+      const end = '2002-11-01T05:00';
+      const expectedStart = DateTime.fromISO(start, dcz).toUTC().toISO();
+      const expectedEnd = DateTime.fromISO(end, dcz).toUTC().toISO();
+      const range = normalizeIsoDateRange(start, end);
+      expect(range.start).toBe(expectedStart);
+      expect(range.end).toBe(expectedEnd);
+    });
+
+    //should throw an exception if given invalid iso start/end dates
+    it('should throw an error if given invalid start', () => {
+      expect(() => {
+        normalizeIsoDateRange('taco tuesday', undefined);
+      }).toThrow();
+
+      expect(() => {
+        normalizeIsoDateRange('today', 'tomorrow');
+      }).toThrow();
+    });
+  });
+
+  describe('calculateDifferenceInHours', () => {
```

Comment:
Might want a test cases covering:
* range timestamps that encompass the DT -> ST time change (extra hour)
* range timestamps that encompass the ST -> DT time change (one hour less)